SourceS/paths.cpp

```diff
@@ -6,13 +6,23 @@
 #include "sdl2_to_1_2_backports.h"
 #endif
 
+#ifndef TTF_FONT_DIR
+#define TTF_FONT_DIR ""
+#endif
+
+#ifndef TTF_FONT_NAME
+#define TTF_FONT_NAME "CharisSILB.ttf"
+#endif

```

Comment:
Why are we using defines for new code? Can't we just use consts instead?

Reply:
This is so that they can overriden via CMake arguments

Reply:
:(

Reply:
its only been moved, but yeah i wouuld also have prefered it being replaced, but i get why that isnt a good option. in some cases.